model_server.py

```
@@ -97,7 +99,7 @@ async def load_models():
 @app.post("/predict", response_model=PredictionResponse)
 async def predict_classification(request: ImageRequest):
     global feature_extractor, classification_model, ev_data
-
+    inputs = feature_extractor(images=image, return_tensors="pt")
```

Ruff (0.11.9)

Undefined name `image`

(F821)